controllers/ndbserver_controller_helpers.go

```diff
@@ -16,18 +16,26 @@ import (
 func getNDBServerDatabasesInfo(ctx context.Context, ndbClient *ndb_client.NDBClient) (databases []ndbv1alpha1.NDBServerDatabaseInfo, err error) {
 	log := log.FromContext(ctx)
 	log.Info("Fetching and converting databases from NDB")
-
-	dbs, err := ndb_api.GetAllDatabases(ctx, ndbClient)
+	databasesResponse, err := ndb_api.GetAllDatabases(ctx, ndbClient)
 	if err != nil {
 		log.Error(err, "NDB API error while fetching databases")
 		return
 	}
+	clonesResponse, err := ndb_api.GetAllClones(ctx, ndbClient)
+	if err != nil {
+		log.Error(err, "NDB API error while fetching clones")
+		return
+	}
+	dbs := append(databasesResponse, clonesResponse...)
```

Comment:
nit: This naming is still a little confusing, we could name it allDbs, DBInfos or smthg?